framework/validators/AtLeastOneValidator.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\validators;
+
+use Yii;
+use yii\base\InvalidConfigException;
+
+/**
+ * AtLeastValidator checks if at least one of many attributes are filled.
+ *
+ * In the following example, the `attr1`, `attr2` and `attr3` attributes will
+ * be verified. If at least one of them are not filled, all will receive an error:
+ *
+ * ~~~[php]
```

Comment:
It is better to use \`\`\`php.

Reply:
not just better, `~~~[php]` does not work as expected.